appstore/apps_core_services/templates/irods_login.html

```diff
@@ -15,4 +15,25 @@
     </form>
 
 </div>
-{% endblock %}
\ No newline at end of file
+<div class="modal fade" id="exampleModal" role="dialog" aria-labelledby="exampleModalLabel"
+     aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-body">
+                <h5>Please check your email for password</h5>
```

Comment:
Can the message be more descriptive? i.e. Please check your email for a password to access iRODS?

Reply:
Yes I will change that as well.